README.md

```diff
@@ -112,6 +112,28 @@ end
 
 Put serializer files in `app/serializers/`. For instance this file should be at `app/serializers/cereal_serializer.rb`.
 
+### Snakecase to lower camel case
+You can pass a method to `transform_keys` in a serializer which will transform the keys for each attribute unless a `from` attribute was specified for that attribute.
+The configuration for `transform_keys` can be inherited, and overridden in the serializers that inherit from another.
+```ruby
+# app/serializers/base_serializer.rb
+class BaseSerializer < CacheCrispies::Base
+  transform_keys lambda { |key| key.to_s.camelize(:lower) } 
```

Comment:
Could we use a block here rather than a lambda argument?

It just reads a bit better and is a bit more consistent to me to have
```ruby
transform_keys lambda { |key| key.to_s.camelize(:lower) }
```

Reply:
It looks like you just repeated the line I have there... accident yes?
You're meaning to have something like this instead?
```ruby
transform_keys do |key|
  key.to_s.camelize(:lower)
end
```